AdyenSession/AdyenSession.swift

```diff
@@ -64,13 +66,16 @@ public final class Session: SessionProtocol {
     /// The presentation delegate.
     public private(set) weak var presentationDelegate: PresentationDelegate?
     
-    /// Initializes an instance of `Session` asynchronously.
+    /// The delegate object.
+    public weak var delegate: AdyenSessionDelegate?
+    
+    /// Initializes an instance of `SessionComponent` asynchronously.
     /// - Parameter configuration: The session configuration.
     /// - Parameter presentationDelegate: The presentation delegate.
     /// - Parameter completion: The completion closure, that delivers the new instance asynchronously.
     public static func initialize(with configuration: Configuration,
                                   presentationDelegate: PresentationDelegate,
```

Comment:
maybe the delegate should be passed here, like the `presentationDelegate` since its required for session to work